deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/TCPClientForMetrics.java

```diff
@@ -0,0 +1,35 @@
+package com.pinterest.teletraan.resource;
+
+import java.io.*;
+import java.net.*;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Calendar;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TCPClientForMetrics {
+    private static final Logger LOG = LoggerFactory.getLogger(TCPClientForMetrics.class);
+    public void emitMetrics(String metricsName) throws Exception {
+        try {
+            String sentence;
+            BufferedReader inFromUser = new BufferedReader(new InputStreamReader(System.in));
+            Socket clientSocket = new Socket("127.0.0.1", 18126);
```

Comment:
Please use a constant for metrics agent port 18126

Reply:
Updated, thanks for reminding.